src/components/NavBar.js

```diff
@@ -0,0 +1,29 @@
+import { FaArrowAltCircleLeft, FaMicrophone } from 'react-icons/fa';
+import { SlSettings } from 'react-icons/sl';
+import { Link } from 'react-router-dom';
+import MealList from '../containers/MealList';
+import '../styles/NavBar.css';
+import '../styles/MealList.css';
+
+const NavBar = () => (
+  <nav data-testid="nav" className="nav">
+    <Link className="back" to="/" onClick={() => MealList()}>
```

Comment:
- [ ] As the previous reviewers mentioned, although, the back button leads to the home page as required, your browser refreshes when the button is clicked and as this is a react project, it is very important that SPA rules are followed. You can read more on [SPA here](https://www.kirupa.com/react/creating_single_page_app_react_using_react_router.htm). To fix this problem kindly use a ```useHistory Hook```as the last reviewer suggested. e.g

``` import React from 'react';
import { Link, useHistory } from 'react-router-dom';
import { FaArrowAltCircleLeft, FaMicrophone, SlSettings } from 'react-icons/all';

const NavBar = () => {
  const history = useHistory();

  const handleBackClick = () => {
    history.goBack();
  };

  return (
    <nav data-testid="nav" className="nav">
      <Link className="back" to="/" onClick={handleBackClick}>
        <FaArrowAltCircleLeft className="icons" />
      </Link>
...
```